site/pages/reference/account-kit/infra/functions/alchemyGasManagerMiddleware.mdx

```diff
@@ -18,13 +18,13 @@ import { alchemyGasManagerMiddleware } from "@account-kit/infra";
 ## Usage
 
 ```ts
-import { sepolia, alchemyGasManagerMiddleware } from "@account-kit/infra";
+import { sepolia, alchemyErc7677Middleware } from "@account-kit/infra";
 import { http } from "viem";
 
 const client = createSmartAccountClient({
   transport: http("rpc-url"),
   chain: sepolia,
-  ...alchemyGasManagerMiddleware("policyId"),
```

Comment:
odd... this probably shouldn't have changed, I wonder what happened here?

oos for this though